### PR TITLE
[ADD] logistic_order_donation_budget

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by lepistone** [comment]
The demo file is not loaded in the module manifest.

**Comment by jgrandguillaume** [comment]
Functionally speaking, it works fine ! :+1:

File: logistic_order_donation_budget/README.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Logistics Order - Donation - Budget
+===================================
```
Comment (yvaucher) [inline logistic_order_donation_budget/README.rst:6]: Can we get a pretty OCA README ?

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**